### ISSUE TITLE
PresentMon Initialization Error - PresentMon service stop on launching 

### ISSUE THREAD

**Issue by alfonslm** [GameTechDev/PresentMon#327]
The PresentMon service stop whenever the application is started and I have tried to repair the installation with the .msi file.

I will assume that there is something with my system after I reinstalled it that causes some sort of issue.
[DxDiag.txt](https://github.com/user-attachments/files/16263122/DxDiag.txt)

**Comment by markgalvan-intel** [comment]
Hello! I'm sorry to hear that the service is having issues. If you have some time would you be able to grab some logs from the service to help us debug the issue. Here is a link for instructions on enable logs in both PresentMonService and Intel PresentMon.

https://github.com/GameTechDev/PresentMon/issues/223#issuecomment-2061636285

Thank you for you time!

**Comment by alfonslm** [comment]
Okay, either I have done something wrong or I got an unexpected result
![image](https://github.com/user-attachments/assets/d38fdaf6-b6f7-4c78-95f2-d01b3cafcd6d)

https://github.com/user-attachments/assets/1ab23085-bdad-4547-b4ae-51948c2047b5

**Comment by alfonslm** [comment]
[cef-debug.log](https://github.com/user-attachments/files/16278499/cef-debug.log)
[log.txt](https://github.com/user-attachments/files/16278500/log.txt)
[log-renderer-18136.txt](https://github.com/user-attachments/files/16278501/log-renderer-18136.txt)

**Comment by markgalvan-intel** [comment]
No, the reason why Intel PresentMon is displaying that message is because the service is crashing. It looks like logs were generated for Intel PresentMon so that's good. Can you also grab log generated by the service? From the video I can see it was enabled in the service properties and it should reside in c:\temp\PM2. Please let us know if the log is not present for the service.

**Comment by alfonslm** [comment]
Well, the folder in the background is the c: so the log didn't get generated there

**Comment by markgalvan-intel** [comment]
Ah. I see what's going on. The folder has to be present in order for the log to be generated. If you create the folder "c:\temp\PM2", then start the service as you did in the video, when the crash occurs the log should be generated.

**Comment by markgalvan-intel** [comment]
Interesting. We have never seen that before. When you have a chance please share the file. Curious what was generated.

**Comment by alfonslm** [comment]
That seems to have been a lot of work for not that much help

[pm-srv-info-20240719-215422.48720.log](https://github.com/user-attachments/files/16316599/pm-srv-info-20240719-215422.48720.log)
```
Log file created at: 2024/07/19 21:54:22
Running on machine: DESKTOP-R53PRK4
Running duration (h:mm:ss): 0:00:00
Log line format: [IWEF]yyyymmdd hh:mm:ss.uuuuuu threadid file:line] msg
I20240719 21:54:22.328909 43728 PMMainThread.cpp:284] Creating comms with NSM name: Global\pm2_bip_shm
```
[cef-debug.log](https://github.com/user-attachments/files/16316612/cef-debug.log)
[log.txt](https://github.com/user-attachments/files/16316613/log.txt)
[log-renderer-18136.txt](https://github.com/user-attachments/files/16316614/log-renderer-18136.txt)
[log-renderer-14640.txt](https://github.com/user-attachments/files/16316615/log-renderer-14640.txt)

**Comment by markgalvan-intel** [comment]
Sorry for all of the trouble. This is actually very useful in our debug efforts as it let's us know where to start the investigation. We very much appreciate your time and effort and will hopefully have a fix soon.

**Comment by alfonslm** [comment]
I just installed the updated version from december and got the same error

**Comment by alfonslm** [comment]
[pm-srv-info-20240719-215422.48720.log](https://github.com/user-attachments/files/18628436/pm-srv-info-20240719-215422.48720.log)
[pmsvc-log-250201-114806.txt](https://github.com/user-attachments/files/18628437/pmsvc-log-250201-114806.txt)

[cef-debug.log](https://github.com/user-attachments/files/18628439/cef-debug.log)
[log.txt](https://github.com/user-attachments/files/18628438/log.txt)

The system should still be the same

**Comment by MayImilae** [comment]
I am also affected by this issue.

**Comment by alfonslm** [comment]
> I am also affected by this issue.

Can you attach the log files and the DxDiag system information?
https://github.com/GameTechDev/PresentMon/issues/223

**Comment by MayImilae** [comment]
The errors are the same, but sure.

[log.txt](https://github.com/user-attachments/files/18631563/log.txt)
[cef-debug.log](https://github.com/user-attachments/files/18631560/cef-debug.log)
[DxDiag.txt](https://github.com/user-attachments/files/18631562/DxDiag.txt)
[pmsvc-log-250202-000021.txt](https://github.com/user-attachments/files/18631561/pmsvc-log-250202-000021.txt)

**Comment by SicSo** [comment]
I have the same issue, was the cause identified?

**Comment by alfonslm** [comment]
> I have the same issue, was the cause identified?

Probably not. It's from a few versions back. You could try 2.0 since I think that I didn't have the issue then but I have not gotten around to test it since it's a hassle to downgrade

**Comment by alfonslm** [comment]
> I have the same issue, was the cause identified?

Also, my guess is that it's an issue with specific systems so can you follow the guide and get the log and system information

**Comment by alfonslm** [comment]
@markgalvan-intel I got it to run but I would just say that it seems wildly inaccurate.

**Comment by alfonslm** [comment]
https://github.com/user-attachments/assets/a5f556df-43a4-418b-a144-56723bcf35fc

**Comment by SicSo** [comment]
> I have the same issue, was the cause identified?

[dxdiag.txt](https://github.com/user-attachments/files/18725266/dxdiag.txt)

[cef-debug.log](https://github.com/user-attachments/files/18725268/cef-debug.log)
[log.txt](https://github.com/user-attachments/files/18725267/log.txt)

[pmsvc-log-250209-163131.txt](https://github.com/user-attachments/files/18725270/pmsvc-log-250209-163131.txt)
[pm-srv-info-20250209-162634.63900.log](https://github.com/user-attachments/files/18725269/pm-srv-info-20250209-162634.63900.log)

**Comment by alfonslm** [comment]
So, all of us are using multiple and different GPU's. IDK if the problem disappear if you disable GPU's until you only have one specific modell running but I could run the program after disabling my 1080. I guess that you two will have to play around a bit with disabling GPU's and see what works and not @SicSo @MayImilae 

It's "definitely" the cause since 2 times is weird and 3 is a pattern

**Comment by markgalvan-intel** [comment]
We looked at all of the provided logs. It appears the code we use to detect the graphics adapters on the machine is failing for some reason. Using the provided dxdiag logs we will attempt to create a machine that reproduces the error. Thank you all for the information provided. We really appreciate the help and support.

**Comment by MayImilae** [comment]
For anyone else who has this issue and found this thread, you can work around it by disabling all but one of your graphics adapters in the device manager.

It's not great but it allows presentmon to work. Tested on 2.3.0.
